src/whitespace.d.ts

```diff
@@ -0,0 +1 @@
+export default function(node: Node, blockTest?: (node: Node) => boolean);
```

Comment:
Nit: there should be a trailing newline.

Would it make more sense to put this in the project root? Then we wouldn't have to mess with npm's `files`.